spawner.sh

```diff
@@ -68,7 +70,7 @@ if [[ -f /opt/git/admin-tools/parse-opt.sh ]]; then
         LOCAL_SSD_TYPE CLUSTER_NAME SITE_CONFIG \
         ES_VERSION PLUGIN_VERSION LOGSTASH_VERSION GITHUB_CREDENTIALS \
         CPU_PLATFORM ES_NODE_CONFIG ES_DOWNLOAD_URL CONTROLLER_IP \
-        CUSTOM_ES_JAVA_OPTS SCOPES DEBUG NUM_SLAVES SLAVE_TYPE; do
+        CUSTOM_ES_JAVA_OPTS USE_BUNDLED_JDK GIT_DEMOS_BRANCH_OPT SCOPES DEBUG NUM_SLAVES SLAVE_TYPE; do
```

Comment:
USE_BUNDLED_JDK is probably better done using a flag rather than a traditional option. Let's discuss...